<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWindow</class>
 <widget class="QWidget" name="EmployeeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="employeeTable">
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Employee Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Employee Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Hired</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contact Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Charge</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="rowToDelete">
     <property name="placeholderText">
      <string>Employee Id to Delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addRow">
     <property name="text">
      <string>Add Row</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeRow">
     <property name="text">
      <string>Remove Row</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loadTable">
     <property name="text">
      <string>Load Database</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
